lib/ansible/modules/database/postgresql/postgresql_sequence.py

```diff
@@ -0,0 +1,646 @@
+      I(newschema) and I(owner).
+    default: present
+    choices: [ absent, present ]
+    type: str
+  data_type:
+    description:
+    - Specifies the data type of the sequence. Valid types are bigint, integer,
+      and smallint. bigint is the default. The data type determines the default
+      minimum and maximum values of the sequence. For more info see the
+      documention
+      U(https://www.postgresql.org/docs/current/sql-createsequence.html).
+    - Supported from PostgreSQL 10.
+    choices: [ bigint, integer, smallint ]
+    type: str
+  increment:
+    description:
+    - Increment specifies which value is added to the current sequence value
+      to create a new value.
+    - A positive value will make an ascending sequence, a negative one a
+      descending sequence. The default value is 1.
+    type: int
+  minvalue:
+    description:
+    - Minvalue determines the minimum value a sequence can generate. The
+      default for an ascending sequence is 1. The default for a descending
+      sequence is the minimum value of the data type.
+    type: int
+    aliases:
+      - min
+  maxvalue:
+    description:
+    - Maxvalue determines the maximum value for the sequence. The default for
+      an ascending sequence is the maximum
+      value of the data type. The default for a descending sequence is -1.
+    type: int
+    aliases:
+      - max
+  start:
+    description:
+    - Start allows the sequence to begin anywhere. The default starting value
+      is I(minvalue) for ascending sequences and I(maxvalue) for descending
+      ones.
+    type: int
+  cache:
+    description:
+    - Cache specifies how many sequence numbers are to be preallocated and
+      stored in memory for faster access. The minimum value is 1 (only one
+      value can be generated at a time, i.e., no cache), and this is also
+      the default.
+    type: int
+  cycle:
+    description:
+    - The cycle option allows the sequence to wrap around when the I(maxvalue)
+      or I(minvalue) has been reached by an ascending or descending sequence
+      respectively. If the limit is reached, the next number generated will be
+      the minvalue or maxvalue, respectively.
+    - If false (NO CYCLE) is specified, any calls to nextval after the sequence
+      has reached its maximum value will return an error. False (NO CYCLE) is
+      the default.
+    type: bool
+  cascade:
+    description:
+    - Automatically drop objects that depend on the sequence, and in turn all
+      objects that depend on those objects.
+    - Ignored if I(state=present).
+    - Only used with I(state=absent).
+    type: bool
+  rename_to:
+    description:
+    - The new name for the I(sequence).
+    - Works only for existing sequences.
+    type: str
+  owner:
+    description:
+    - Set the owner for the I(sequence).
+    type: str
+  schema:
+    description:
+    - The schema in the new I(sequence) will be created.
+    default: public
+    type: str
+  newschema:
```

Comment:
Why have both `schema` and `newschema`? I would assume that if I specify another value for `schema`, that the schema will be changed.

Reply:
Yes and no ;)
If you specify a sequence in an other schema then public, it will be difficult to find this sequence if you use only `schema` for the new schema name. Therefor I would prefer to use `newschema` for setting a new schema to a existing sequence.
Are you okay with that?

Reply:
Sure. But then you should really adjust the documentation, because right now it seems that `schema` is only needed for creation, not for actually finding back the sequence.